### PR TITLE
Move implicit pull test to use local registry

### PR DESCRIPTION
**Pull request by tonistiigi**
The test added in #22247 is failing in the arm test suite https://jenkins.dockerproject.org/job/Docker%20Master%20(arm)/1620/console because it pulls and tries to run amd64 image. This moves the test to a local registry.

@vdemeester 

Signed-off-by: Tonis Tiigi tonistiigi@gmail.com

### CODE REVIEW

File: integration-cli/docker_cli_pull_local_test.go [hunk 1]
```diff
@@ -421,3 +421,26 @@ func (s *DockerRegistryAuthHtpasswdSuite) TestPullWithExternalAuth(c *check.C) {
 
 	dockerCmd(c, "--config", tmp, "pull", repoName)
 }
+
+// TestRunImplicitPullWithNoTagOnlyPullDefaultTag should pull implicitely only the default tag (latest)
+func (s *DockerRegistrySuite) TestRunImplicitPullWithNoTagOnlyPullDefaultTag(c *check.C) {
```
Comment (aaronlehmann) [inline integration-cli/docker_cli_pull_local_test.go]: I know you are only moving the test, but can you shorten the name to `TestRunImplicitPullWithNoTag`?

**Comment by cpuguy83** [comment]
LGTM

**Comment by tonistiigi** [comment]
Renamed the test. I'm not going to start the arm suite again, the test passed in previous run https://jenkins.dockerproject.org/job/Docker-PRs-arm/261/console

**Comment by aaronlehmann** [comment]
LGTM

**Comment by vdemeester** [comment]
Ah right, didn't thought of that :sweat:

**Pull request merged**